### DISCUSSION TITLE
When was the first commercial text message sent?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#894] [Galaxy Trivia] [answered]
**Bonus:** What did it say?

**Comment by Carol42** [comment, marked as answer]
The first SMS message was sent on 3rd December 1992, when Neil Papworth, a test engineer for Sema Group, sent "Merry Christmas" to the Orbitel 901 phone of colleague Richard Jarvis.

**Comment by peterbaccaro** [comment]
The first commercial text message was sent on **December 3, 1992**. It was sent by Neil Papworth, a 22-year-old British software programmer, to Richard Jarvis, a director at Vodafone UK. The message was sent from a computer to Jarvis' Orbitel 901 mobile phone, and it simply said "**Merry Christmas**". This marked the beginning of a new era in mobile communication, as text messaging quickly gained popularity and became a widespread means of communication around the world.

**Comment by tshenolo** [comment]
It was sent on December 3, 1992, 
Contents where "Merry Christmas."
It was sent by Neil Papworth, a 22-year-old British software engineer, to Richard Jarvis of Vodafone
The message was sent from a computer to Jarvis's Orbitel 901 mobile phone.
